src/index.js

```diff
@@ -88,6 +92,9 @@ export default function alias(options = {}) {
         return filePath + '.js';
       }
 
+      const nodeModuleFile = resolveNodeModules(updatedId);
+      if (nodeModuleFile) return nodeModuleFile;
```

Comment:
So, you can write a alias like this:
`rollup({
  entry: './src/index.js',
  plugins: [alias({
    vue: 'vue/dist/vue.esm.js'
  })],
});`
to include the runtime template compiler in vue project.